datasketches/tests/frequencies_serialization_test.rs

```diff
@@ -128,7 +128,11 @@ fn test_cpp_frequent_longs_compatibility() {
         let sketch = FrequentItemsSketch::<i64>::deserialize(&bytes, serde);
         if cfg!(windows) {
             if let Err(err) = sketch {
-                assert!(matches!(err, SerdeError::InsufficientData(_)));
+                assert_eq!(err.kind(), ErrorKind::InvalidData);
+                assert!(
```

Comment:
nit: googletest has a contains matcher that can provide a better error message without manual construct.

https://docs.rs/googletest/latest/googletest/matchers/fn.contains.html